apps/marketing/src/components/(marketing)/single-player-mode/create-single-player-document.action.ts

```diff
@@ -177,9 +177,7 @@ export const createSinglePlayerDocument = async (
     },
   );
 
-  // Todo: Handle `downloadLink`
   const template = createElement(DocumentSelfSignedEmailTemplate, {
-    downloadLink: `${process.env.NEXT_PUBLIC_MARKETING_URL}/single-player-mode/${documentToken}`,
```

Comment:
We still want the download link I think 🤔

Reply:
How come? The "download" link just takes them back to the SPM success page

The PDF is now directly attached to the email, so they can directly view/download it

Reply:
Yep you're right I'm the silly one